app/api/v0/proofs/proved/route.ts

```diff
@@ -129,7 +129,7 @@ export const POST = withAuth(async ({ request, user, timestamp }) => {
     program_id: programId,
     proof_status: "proved",
     proved_timestamp: timestamp,
-    size_bytes: Buffer.byteLength(proofHex, "hex"),
+    size_bytes: binaryBuffer.byteLength,
```

Comment:
Great... this logic looks like it should work to calculate the appropriate byte length when testing locally; clean as well